zerver/migrations/0396_alter_realm_want_advertise_in_communities_directory.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.13 on 2022-05-30 09:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0395_alter_realm_org_type'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='realm',
+            name='want_advertise_in_communities_directory',
+            field=models.BooleanField(db_index=True, default=True),
+        ),
+    ]
```

Comment:
Probably easier for this hack to just change the query to skip checking `want_advertise_in_communities_directory`.